packages/apps/human-app/frontend/src/styles/dark-color-palette.ts

```diff
@@ -45,7 +45,7 @@ export const darkColorPalette = {
     main: 'rgba(203, 207, 232, 0.28)',
   },
   button: {
-    disabled: 'rgba(218, 222, 240, 0.8)',
+    disabled: '#FFFFFF1F',
```

Comment:
It's still different from the color framework in both modes

What we should have:
<img width="276" alt="Screenshot 2024-10-28 at 13 35 55" src="https://github.com/user-attachments/assets/72490b2b-a30d-47ea-8d46-fda649462d42">
What we actually have
<img width="133" alt="Screenshot 2024-10-28 at 13 36 37" src="https://github.com/user-attachments/assets/9460defa-ba7e-4ff8-8613-1b89ff181b58">

<img width="247" alt="Screenshot 2024-10-28 at 13 36 47" src="https://github.com/user-attachments/assets/37ef831c-6211-4639-b68e-57d50f23cf59">

Applies to all button variants